core-libs/setup/ssr/logger/transformers/utils/transformers-utils.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TransformerConfig } from '../default-transformers';
+import { ExpressLogTransformer } from '../express-log-transformer';
+
+export interface ExpressLogTransformerConfig {
+  replace?: TransformerConfig;
+  remove?: [keyof TransformerConfig];
```

Comment:
Instead of introducing a custom convention with a configurable `replace/remove` did you consider one of those 2 alternatives:
a) transformers being simply an array
b) transformers being an key-object map. we could use `deepMerge() from @spartacus/core` to reset/overwrite default object's keys.